templates/server-acl-init-job.yaml

```diff
@@ -55,7 +55,10 @@ spec:
                 -create-inject-token=true \
                 {{- end }}
                 {{- if .Values.connectInject.aclBindingRuleSelector }}
-                -acl-binding-rule-selector={{ .Values.connectInject.aclBindingRuleSelector }}
+                -acl-binding-rule-selector={{ .Values.connectInject.aclBindingRuleSelector }} \
+                {{- end }}
```

Comment:
Try this with a binding selector configured but no licence. The stray continuation worries me

Reply:
@rboyer I think it ends up being okay because there's a blank line at the end of the command. Here's the output from `kubectl describe pod`:

```
    Command:
      /bin/sh
      -ec
      consul-k8s server-acl-init \
        -release-name=consul \
        -k8s-namespace=default \
        -expected-replicas=1 \
        -allow-dns=true \
        -create-inject-token=true \
        -acl-binding-rule-selector=serviceaccount.name!=default \
      
```

The job appears to run okay, but I agree, the extra backslash looks wrong.

Reply:
I haven't had any issue with it, but switching it up just in case.